PendingReleaseNotes

```diff
@@ -90,6 +90,11 @@
   redistribution. In contrast, a stretch rule with a single-take configuration
   will not cause any data movement during the upgrade process.
 
+* CephFS: Command "ceph fs volume create" now allows users to pass metadata
+  and data pool names that should be used for creating the volume. In case
+  the passed name is a name of a non-empty pool, the command will abort.
+
```

Comment:
Don't I recall reviewing this change a couple of days ago in another PR?

Suggest:

* CephFS: The `ceph fs volume create` command now allows admins to supply metadata
   and data pool names to be used when creating the volume. If any of these pools
  exists and is non-empty pool, the command will abort.

Reply:
I guess you missed my  reply. There are 3 separate features in PR and therefore clubbing all of them into one release note is not a good idea. Plus, review only latter 2 features, there is a separate PR for former feature. For more details please check earlier replies.